gcloud/bigtable/cluster.py

```diff
@@ -15,9 +15,51 @@
 """User friendly container for Google Cloud Bigtable Cluster."""
 
 
+import re
+
 from gcloud.bigtable.table import Table
 
 
+_CLUSTER_NAME_RE = re.compile(r'^projects/(?P<project>[^/]+)/'
+                              r'zones/(?P<zone>[^/]+)/clusters/'
+                              r'(?P<cluster_id>[a-z][-a-z0-9]*)$')
+
+
+def _require_pb_property(message_pb, property_name, value):
```

Comment:
Should `value` default to None, to enable the "just give me the current value" semantics, which are all we are using at the moment?